packages/cdk/lib/util/instance-types.ts

```diff
@@ -190,3 +194,11 @@ const isInstanceTypeSupported = (instanceType: string, region?: string): boolean
   }
   return true;
 };
+
+const isInstanceTypeArmBased = (instanceType: string): boolean => {
+  const instanceTypeFamily = instanceType.split(".")[0];
```

Comment:
You should use CDK's logic for this since it's more likely to handle all current and feature ARM architectures better

https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-ec2.InstanceType.html#architecture

Reply:
Thanks. Will do.